ext/odbc/config.m4

```diff
@@ -357,23 +357,17 @@ fi
 
 if test -z "$ODBC_TYPE"; then
 PHP_ARG_WITH([unixODBC],,
-  [AS_HELP_STRING([[--with-unixODBC[=DIR]]],
-    [Include unixODBC support [/usr/local]])])
+  [AS_HELP_STRING([--with-unixODBC],
+    [Include unixODBC support])])
 
-  AC_MSG_CHECKING(for unixODBC support)
+  AC_MSG_CHECKING(whether to build with unixODBC support)
   if test "$PHP_UNIXODBC" != "no"; then
-    if test "$PHP_UNIXODBC" = "yes"; then
-      PHP_UNIXODBC=/usr/local
-    fi
-    ODBC_INCDIR=$PHP_UNIXODBC/include
-    ODBC_LIBDIR=$PHP_UNIXODBC/$PHP_LIBDIR
-    ODBC_LFLAGS=-L$ODBC_LIBDIR
-    ODBC_CFLAGS=-I$ODBC_INCDIR
-    ODBC_LIBS=-lodbc
+    AC_MSG_RESULT(yes)
+    PKG_CHECK_MODULES([ODBC], [odbc])
+    PHP_EVAL_INCLINE($ODBC_CFLAGS)
     ODBC_TYPE=unixODBC
-    PHP_ODBC_CHECK_HEADER(sqlext.h)
+    AC_CHECK_HEADER(sqlext.h,, [AC_MSG_ERROR(ODBC header file sqlext.h not found.)])
```

Comment:
Is this check still necessary if we're using pkg-config?

Reply:
@nikic, I only included the header check to maintain the original behaviour. That said, it’s not needed now.

pkg-config lets us know unixodbc-dev or some similar package is installed. I think it’s safe to trust that all required headers are present, since PKG_CHECK_MODULES will fail if odbc.pc is not found (i.e. the development package is not installed).

I’ll update the PR to remove the header check, unless you disagree?

Reply:
Yeah, I'd prefer to drop the header check. I think generally we should trust pkg-config and only perform additional checks if we need to detect differences between versions / supported features or similar.